WordPress/UITestsFoundation/FancyAlertComponent.swift

```diff
@@ -23,7 +23,8 @@ public class FancyAlertComponent: ScreenObject {
     public init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
             expectedElementGetters: [defaultAlertButtonGetter, cancelAlertButtonGetter],
-            app: app
+            app: app,
+            waitTimeout: 3
```

Comment:
The default timeout for `ScreenObject` is 20s (which might not be a good idea)... This view is one we check for often: we don't want to wait 20s for it every time.